projects/Rockchip/devices/RK3288/linux/default/linux.arm.conf

```diff
@@ -5752,7 +5752,7 @@ CONFIG_GENERIC_PHY=y
 CONFIG_PHY_ROCKCHIP_DP=m
 # CONFIG_PHY_ROCKCHIP_DPHY_RX0 is not set
 CONFIG_PHY_ROCKCHIP_EMMC=m
-CONFIG_PHY_ROCKCHIP_INNO_HDMI=m
+# CONFIG_PHY_ROCKCHIP_INNO_HDMI is not set
```

Comment:
Is this intentional in this commit? You only talk about enabling USB.

Reply:
Yes.
Inno HDMI phy is only used on RK322x and RK3328 and wouldn't want to make an extra commit for every change